### PR TITLE
Fix blacklist task bug

### PR DESCRIPTION
**Pull request by dseisun**
Fixes a bug where the first blacklisted workout  encountered would just
close out the category entirely instead of just skipping the workout

### CODE REVIEW

**Comment by dseisun** [comment]
goddamnit, knew I should have gotten you to review @jnguyen75

**Comment by jnguyen75** [comment]
haha not sure I would have caught that though.

File: workout.py [hunk 1]
```diff
@@ -74,10 +74,10 @@ def generate_exercises_for_category(self,
         cat_excercises = self.shuffled_grouped_exercises[cat.id]
         for i in count():
             exc = cat_excercises[i % len(cat_excercises)]
-            if category_time > exc.time and exc.id not in set(
-                         map(lambda exc: exc.id, blacklisted_exercises)):
-                exc_list.append(exc)
-                category_time -= exc.time
+            if category_time > exc.time:
+                if exc.id not in set(map(lambda exc: exc.id, blacklisted_exercises)):
```
Comment (jnguyen75) [inline workout.py]: why do you need a set here?

Comment (dseisun) [reply workout.py]: testing for inclusion (`not in`) only works on a set

**Pull request merged**